helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/constraints/PartitionMovementConstraint.java

```diff
@@ -38,59 +37,55 @@
  * evaluated score will become lower.
  */
 class PartitionMovementConstraint extends SoftConstraint {
-  private static final float MAX_SCORE = 1f;
-  private static final float MIN_SCORE = 0f;
-  //TODO: these factors will be tuned based on user's preference
+  // TODO: these factors will be tuned based on user's preference
   // This factor indicates the default score that is evaluated if only partition allocation matches
   // (states are different).
-  private static final float ALLOCATION_MATCH_FACTOR = 0.5f;
   // This factor indicates the contribution of the Baseline assignment matching to the final score.
-  private static final float BASELINE_MATCH_FACTOR = 0.25f;
+  private float _baseLineMatchFactor = 0.25f;
```

Comment:
Could you please add a little more explanation about this? What does it mean for baselinematchfactor to be 0? What does it mean for it to be 1.0f?

Reply:
The factor, in general, defines the goodness of the assignment if it matches with baseline assignment; Actually, it shouldn't be part of the PR (it's more about refining the existing partition movement) so I reverted this part of changes. If in the next PR of refining the implementation, I will address it and add more comments.